src/main/java/thaumicboots/api/ItemBoots.java

```diff
@@ -126,7 +126,20 @@ public boolean getIsRepairable(ItemStack par1ItemStack, ItemStack par2ItemStack)
 
     protected float computeBonus(ItemStack itemStack, EntityPlayer player) {
         int ticks = player.inventory.armorItemInSlot(0).stackTagCompound.getInteger("runTicks");
-        return runBonus + ((ticks * 0.2F) * longrunningbonus);
+        return speedBonus + ((ticks * 0.2F) * longrunningbonus);
+    }
+
+    protected boolean checkNanoChestplate(EntityPlayer player) {
```

Comment:
The reason this check is needed is because `player.capabilities.isFlying` is not true when in hover mode with this chestplate? What about others, like IC2 quantum, or Gravi?

Reply:
IC2 Quantum seems to not be a flight armor. It works with the gravichestplate though. That one seems to set `player.isFlying` to `true`